src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -2479,8 +2479,7 @@ def test_aks_azure_service_mesh_get_upgrades(self, resource_group, resource_grou
 
     @AllowLargeResponse()
     @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
-    @AKSCustomRoleBasedServicePrincipalPreparer()
-    def test_aks_create_with_standard_sku(self, resource_group, resource_group_location, sp_name, sp_password):
+    def test_aks_create_with_standard_sku(self, resource_group, resource_group_location):
```

Comment:
This test case does not actually use the service principal, so the unused variable and the resource preparer are deleted.

BTW, queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93310143&view=results) for these 2 cases.
- test_aks_create_with_standard_sku
-test_aks_update_with_windows_password